GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift

```diff
@@ -48,7 +48,7 @@ final class WebsiteAnalyticsWidget: WidgetControlling {
     }
 
     private func updateWithReports(reports: [PageViewsRowReport]) {
-        let reportsSlice = reports[0...min(3, reports.count)]
+        let reportsSlice = reports.count > 0 ? reports[0..<min(3, reports.count)] : []
```

Comment:
You don't need ternary operator here. `0..<0` is just an [empty range](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html) and results in empty array.

```swift
let slicedReports = reports[0..<min(3, reports.count)]
```